controls/memcached_habservice_works.rb

```diff
@@ -0,0 +1,38 @@
+title 'Tests to confirm memcached habitat service works as expected'
+
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input('plan_name', value: 'memcached')
+service_port=11211
```

Comment:
For convenience, let's maybe make this an input with default value of 11211.

Reply:
Done.  Thanks.